src/domain/auth/LoginForm.tsx

```diff
@@ -0,0 +1,48 @@
+import { Form, useNavigation } from 'react-router-dom'
+import { redirect } from 'react-router-dom'
+import { toast } from 'react-toastify'
+import Input from '../../components/Input/Input'
+import styles from './../../pages/Login/Login.module.css'
+
+export const action = async ({ request }) => {
+  // what register form needs to do
+
+  const formData = await request.formData()
+  const data = Object.fromEntries(formData)
+
+  const { email, password } = data
```

Comment:
From our conversation
```suggestion
  const email = formData.get("email")
  const password = formData.get("password")
```